src/analyserqueue.cpp

```diff
@@ -192,10 +191,10 @@ bool AnalyserQueue::doAnalysis(TrackPointer tio, Mixxx::AudioSourcePointer pAudi
             // Complete analysis block of audio samples has been read.
             QListIterator<Analyser*> it(m_aq);
             while (it.hasNext()) {
-                Analyser* an =  it.next();
-                //qDebug() << typeid(*an).name() << ".process()";
+                Analyser* an = it.next();
+                // qDebug() << typeid(*an).name() << ".process()";
```

Comment:
This is the fault we have already discussed. Can we allow both in the clang format file?